src/methods/area.jl

```diff
@@ -120,18 +120,28 @@ to be closed.
 =#
 function _signed_area(::Type{T}, geom) where T
     area = zero(T)
-    # Close curve, even if last point isn't explicitly repeated
     np = GI.npoint(geom)
     np == 0 && return area
-    first_last_equal = equals(GI.getpoint(geom, 1), GI.getpoint(geom, np))
-    np -= first_last_equal ? 1 : 0 
+
+    first = true
+    local pfirst, p1
```

Comment:
What does this do? Just for my future knowledge?

Reply:
It makes the variables available in the scope outside thep loop